trillian_map_api.proto

```diff
@@ -137,6 +137,25 @@ message SetMapLeavesResponse {
   SignedMapRoot map_root = 2;
 }
 
+message WriteMapLeavesRequest {
+  int64 map_id = 1;
+  // The leaves being set must have unique Index values within the request.
+  repeated MapLeaf leaves = 2;
+  bytes metadata = 3;
```

Comment:
Worth a comment on `metadata`? Presumably this is popped into the SMR?

Reply:
Done.